modules/core/src/main/kotlin/datamaintain/core/step/executor/Executor.kt

```diff
@@ -73,8 +74,13 @@ class Executor(private val context: Context) {
                 logger.info {"start marking script ${script.name} as executed"}
                 val executedScript = ExecutedScript.build(script, Execution(ExecutionStatus.OK), context.config.flags)
 
-                markAsExecuted(executedScript)
-                if (!context.config.porcelain) { logger.info { "${executedScript.name} only marked as executed (so not executed)" } }
+                try {
+                    markAsExecuted(executedScript)
+                    if (!context.config.porcelain) { logger.info { "${executedScript.name} only marked as executed (so not executed)" } }
+                } catch (e: DatamaintainQueryException) {
+                    logger.warn { "Failed to mark script $executedScript as executed. Use the following command to force mark the script as executed : " +
```

Comment:
Printing the whole script may be a little bit too ambitious. Maybe the path to the file would be enough? Or just the name?